src/appmetrics.cpp

```diff
@@ -714,22 +736,64 @@ void init(Local<Object> exports, Local<Object> module) {
     /*
      * Set exported functions
      */
-    exports->Set(Nan::New<String>(asciiString("getOption")).ToLocalChecked(), Nan::New<FunctionTemplate>(getOption)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("setOption")).ToLocalChecked(), Nan::New<FunctionTemplate>(setOption)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("start")).ToLocalChecked(), Nan::New<FunctionTemplate>(start)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("spath")).ToLocalChecked(), Nan::New<FunctionTemplate>(spath)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("stop")).ToLocalChecked(), Nan::New<FunctionTemplate>(stop)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("localConnect")).ToLocalChecked(), Nan::New<FunctionTemplate>(localConnect)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("nativeEmit")).ToLocalChecked(), Nan::New<FunctionTemplate>(nativeEmit)->GetFunction());
-    exports->Set(Nan::New<String>(asciiString("sendControlCommand")).ToLocalChecked(), Nan::New<FunctionTemplate>(sendControlCommand)->GetFunction());
+    Isolate* isolate = v8::Isolate::GetCurrent();
+    Local<Context> context = Nan::GetCurrentContext();
+    exports->Set(
+        Nan::New<String>(asciiString("getOption")).ToLocalChecked(),
+        Nan::New<FunctionTemplate>(getOption)
+            ->GetFunction(context).ToLocalChecked()
+	);
```

Comment:
All of these `exports->Set()` can be replaced with the following pattern:
```suggestion
    Nan::SetMethod(exports, asciiString("getOption"), getOption);
```